### PR TITLE
User guide for testing toolkit.

### PR DESCRIPTION
**Pull request by HenkKodde**
Created the "Monitoring section" for the users guide.

### CODE REVIEW

File: documents/User-Guide.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    3.6 [Callback Rules (Success Callbacks)](#36-callback-rules)
+
+#### 3.7 Outbound Request
+[TODO - complete this section of user-guide]
+
+#### 3.8 Settings
+
+### 1. Introduction
+
+The intention of this document is to provide a basic user guide to the mojaloop Self Testing Toolkit. This easy to use tool set was designed for both technical and non-technical users, even though the primarily users would likely be Quality Assurance (QA) resources. By following this guide, users will have a better understand of the capabilities and fuctionality of the tool set.
```
Comment (vijayg10) [inline documents/User-Guide.md]: I think it should be toolkit

File: documents/User-Guide.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+More information would be disclosed once the developement is at a more mature stage.
+
+#### 3.3 Monitoring
+
+Allows you to monitor the incomming and outgoing request.
+
+![Monitoring Initial State](/assets/images/Monitoring_Initial_State.png) 
+
+If you have followed the installation in structions, you should have the Simulator UI open in a browser tab. Go to the **Outbound Send** tab on the left.
```
Comment (vijayg10) [inline documents/User-Guide.md]: *in structions

File: documents/User-Guide.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+Press **Send Transfer** button to send a request from the Simulator UI to the Self Testing Toolkit.
+
+![Send Transfer](/assets/images/Send_Transfer.png)
+
+
+If the request was send successfully from the Simulator UI.
```
Comment (vijayg10) [inline documents/User-Guide.md]: *was send

File: documents/User-Guide.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+![Simulator response](/assets/images/Simulator_Response.png) 
+
+Go back to the Monitoring tab on the Self Testing Toolset. You will notice the three processes assosiated with this request in acending order.
```
Comment (vijayg10) [inline documents/User-Guide.md]: three processes assosiated with this request in acending order -> three requests associated with a transfer in an order such that most recent request first

File: documents/User-Guide.md [hunk 5]
```diff
@@ -18,6 +18,23 @@
 
     3.4 [Sync Response Rules](#34-sync-response-rules)
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (HenkKodde) [reply documents/User-Guide.md]: Have resolved all the conflicts. Branch is ready to be merged.

**Pull request closed without merging**